docs/pt/docs/tutorial/body-fields.md

```diff
@@ -0,0 +1,48 @@
+# Body - Fields
+
+Da mesma forma que voĉe pode declarar uma validação adicional e metadata nas *funções de operação de caminho* com `Query`, `Path` e `Body`, você pode declarar validação e metadata dentro de modelos Pydantic usando `Field` do Pydantic.
```

Comment:
I believe we're using "rota" instead of "caminho", not sure tho.
```suggestion
Da mesma forma que voĉe pode declarar uma validação adicional e metadados nas *funções de operação de rota* com `Query`, `Path` e `Body`, você também pode declarar validações e metadados dentro dos modelos Pydantic usando `Field` do Pydantic.
```

Reply:
I used the ´caminho´ because in the other parts of the translation already using the 'caminho' instead of 'rotas'

Reply:
If you search for `rotas` in the main page, you'll see it's there: https://fastapi.tiangolo.com/pt/